src/index.html

```diff
@@ -5,6 +5,7 @@
     <link href="/fav.ico" rel="shortcut icon">
     <!-- <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> -->
     <meta name = "viewport" content = "width = device-width">
+    <script src=https://embed.runkit.com/ async></script>
```

Comment:
@wzalazar Is there a correct way to load this script that will work with our config?

Reply:
I believe this external script now opens up our sessionStorage user session token up for XSS. #318 

Is it possible to run this script via webpack? There is a script loader plugin for weback that allows us to save this JS as a file and then run it globally via webpack.

https://github.com/webpack-contrib/script-loader

Reply:
This is not a huge deal, as our session don't last that long and the token is now in sessionStorage so don't waste a lot of time on trying the script loader.

Reply:
Let's try to bundle it if possible/simple. This can be a separate PR though.

Reply:
we have to set this security to load external files Subresource Integrity
https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity

Reply:
I wasn't able to figure out the script-loader for webpack, but I believe that I did everything set up with Subresource Integrity. @kennylavender @wzalazar @geoffturk